apps/trackers/jira/query.py

```diff
@@ -868,6 +874,21 @@ def generate_embargo_status(self):
 
         self._query["fields"][field_id] = {"value": choice_str}
 
+    def generate_only_embargo_and_security_level(self):
+        """
+        Generate only the embargo status and security level fields.
+
+        The overwriting of _query, which initializes to None,
+        prevents class TrackerQueryBuilder's accessor method `query` from
+        generating the entire query, otherwise it will call `generate`
+        if `_query` is None.
+        """
+
+        self.generate_base()
+        self._query["fields"]["security"] = None
```

Comment:
This is not necessary since you are now generating the base on the previous line,  `generate_security` will eventually se this if necessary.